tests/testing_common.py

```diff
@@ -561,6 +562,48 @@ def _test_merge_layers(self, model_id, config_cls, config_kwargs):
         logits_merged_from_pretrained = model_from_pretrained(**dummy_input)[0]
         self.assertTrue(torch.allclose(logits_merged, logits_merged_from_pretrained, atol=atol, rtol=rtol))
 
+    def _test_merge_layers_multi(self, model_id, config_cls, config_kwargs):
```

Comment:
The test was added but is not called in `test_decoder_models.py` etc.

Reply:
AH yes nice catch, fixed it!